quickcheck/tasty-quickcheck.cabal

```diff
@@ -29,7 +29,7 @@ library
   other-extensions:    GeneralizedNewtypeDeriving, DeriveDataTypeable
   build-depends:       base >= 4.8 && < 5,
                        tagged < 0.9,
-                       tasty >= 1.5 && < 1.6,
+                       tasty >= 1.5.1 && < 1.6,
```

Comment:
The change does what I need, thanks!

... but

The `tasty-1.5.1` on Hackage has `base <0` revision made.

If I use both `core` and `quickcheck` from the repo, then `cabal-install` finds an install plan.

Reply:
That's expected, I'll release `tasty-1.5.1.1` (or `1.5.2`) soon.